tox.ini

```diff
@@ -0,0 +1,24 @@
+# For more information about tox, see https://tox.readthedocs.io/en/latest/
+[tox]
+envlist = py36,py37,pypy3,black
```

Comment:
Please add `docs` env as well for testing if building documentation isn't broken.